package.json

```diff
@@ -11,67 +11,93 @@
     "SMART ASCVD risk app"
   ],
   "scripts": {
-    "start": "webpack-dev-server --config webpack.dev.config.js --inline",
+    "build": "npm run clean && webpack --config webpack.prod.config.js --progress --colors --display-error-details",
     "clean": "rimraf build",
-    "test": "jest; mocha --require tests/helpers/browser.js tests/app/*.js",
-    "test:nightwatch": "WEBPACK_CONFIG_PATH=../../../tests/features/config.js node ./node_modules/terra-toolkit/lib/scripts/nightwatch-process.js --config tests/nightwatch.conf.js --env default-tiny,default-small,default-medium,default-large,default-huge,default-enormous",
     "lint": "eslint app/*.js components/**/**/*.jsx views/**/index.jsx components/**/*.jsx --config .eslintrc --fix",
-    "build": "npm run clean && webpack --config webpack.prod.config.js --progress --colors --display-error-details"
+    "start": "webpack-dev-server --config webpack.dev.config.js --inline",
+    "test": "jest; mocha --require tests/helpers/browser.js tests/app/*.js",
+    "test:wdio": "wdio tests/wdio.conf.js"
   },
   "publishConfig": {
     "registry": "https://registry.npmjs.org/"
   },
   "dependencies": {
     "canadarm": "^1.0.2",
     "classnames": "^2.2.5",
-    "jquery": "^3.1.1",
+    "intl": "^1.2.5",
+    "jquery": "^3.4.1",
     "react": "^15.4.1",
     "react-dom": "^15.4.1",
-    "react-intl": "^2.2.3",
-    "terra-clinical-error-view": "^1.3.0",
-    "intl": "^1.2.5"
+    "react-intl": "^2.9.0",
+    "terra-clinical-error-view": "^1.3.0"
   },
   "devDependencies": {
-    "babel-core": "^6.18.2",
-    "babel-jest": "^17.0.2",
-    "babel-loader": "^6.2.8",
+    "@babel/cli": "^7.6.4",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
```

Comment:
Are all of these @babel/plugin-proposal-* modules needed?

Reply:
I think so. I used https://github.com/babel/babel-upgrade to go through it.

Reply:
+1